rest/route/agent_test.go

```diff
@@ -820,3 +820,68 @@ func TestCreateGitHubDynamicAccessToken(t *testing.T) {
 		})
 	}
 }
+
+func TestRevokeGitHubDynamicAccessToken(t *testing.T) {
+	route := "/task/%s/github_dynamic_access_token"
+	taskID := "taskID"
+	token := "some-token-would-be-here"
+	json, err := json.Marshal(token)
```

Comment:
nit: not a big deal, but I generally avoid naming my local vars the same as the package itself because it makes it difficult if someone actually wants to use that package elsewhere in the function. Like for tasks, I always call my variable `t` or `tsk` instead of `task` to avoid ambiguity.